app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :count_post
+
```

Comment:
- [ ] As part of this module requirement, _Post_ model should have a method that returns the **5 most recent comments for a given post**. This should be similar to:
```ruby
# app/models/post.rb
...

def recent_comments
  comments.order('created_at Desc').limit(5)
end
```

Reply:
Thank You for the suggestion. I've implemented the requested changes.